cli/src/main/scala/dev/chronicles/cli/Console.scala

```diff
@@ -10,11 +12,19 @@ import scala.io.StdIn
 trait Console[F[_]] {
   def println(msg: String): F[Unit]
   def errorln(msg: String): F[Unit]
+
+  def printlns(msgs: Stream[F, String])(implicit F: Functor[F]): Stream[F, String] =
+    msgs.evalTap(println)
+
+  def errorlns(msgs: Stream[F, String])(implicit F: Functor[F]): Stream[F, String] =
+    msgs.evalTap(println)
```

Comment:
Ah good catch! Also clearly shows I've skimped on tests here... 😱

Interesting point about `final`, I guess that would make sense.

As for whether to return streams of `Unit` or `String`s, I guess it's all a bit speculative at the moment. I thought it might be useful at some point to pipe things through the console and then on to something else, but that's probably a case of YAGNI.

Reply:
I removed `errorlns` as it wasn't used, and went with `Stream[F, Unit]` for the return type of `printlns`.